### PR TITLE
Accessibility updates

### PR DESCRIPTION
**Pull request by Doniee**
- Adding `id` to city detail table headings and a matching `aria-labelledby` to tables
- Make active filter indicator a div instead of button. It looks like the filter buttons but it doesn't do anything, so div is better.

### CODE REVIEW

File: src/frontend/main/MapView.tsx [hunk 1]
```diff
@@ -123,11 +122,14 @@ const FilterWrapper = styled.div<FilterWrapperProps>`
   }
 `;
 
-const ActiveFilter = styled(PrimaryButton)`
+const ActiveFilter = styled.div`
```
Comment (nathandao) [inline src/frontend/main/MapView.tsx:125]: :+1:  nice!

File: src/frontend/main/CityTables.tsx [hunk 2]
```diff
@@ -178,7 +178,7 @@ const CityTables = ({ data, selectedCity, isEmbed }: CityTablesProps) => {
               )}
             </CityHeadingContainer>
             {formattedData.responsesTotal != null && (
-              <Table>
+              <Table aria-labelledby={`${item.city}-table-heading`}>
```
Comment (nathandao) [inline src/frontend/main/CityTables.tsx:181]: Cool! I didn't know such aria attribute exists : D

File: src/frontend/main/MapView.tsx [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+  const handleFilterChange = (filterName: string) => {
+    setSelectedFilter(filterName as FilterKey);
+  };
+
+  const totalResponses = data.reduce((accumulator: number, currentValue: any) => {
+    if (currentValue.responses < 0) return accumulator;
+    return accumulator + currentValue.responses;
+  }, 0);
+
+  window.addEventListener('resize', () => {
```
Comment (cutepig) [inline src/frontend/main/MapView.tsx:228]: Should this be put inside `useEffect` with call to `removeEventListener` in the cleanup function?

Comment (nathandao) [reply src/frontend/main/MapView.tsx]: Looks like this is fixed in master https://github.com/futurice/symptomradar/blob/master/src/frontend/main/map/MapView.tsx#L120 , probably after @Doniee 's route structure refactor? :thinking: 

Nice catch nevertheless :+1: 

I rebased master to resolve some merge conflicts and this file is moved to `src/frontend/main/map/MapView.tsx` now : D

**Pull request merged**